NumLib/DOF/LocalToGlobalIndexMap.cpp

```diff
@@ -272,6 +272,37 @@ LocalToGlobalIndexMap::getNumberOfElementDOF(std::size_t const mesh_item_id) con
     return ndof;
 }
 
+std::size_t
+LocalToGlobalIndexMap::getNumberOfElementComponents(std::size_t const mesh_item_id) const
+{
+    std::size_t n = 0;
+    for (unsigned c=0; c<_rows.cols(); ++c)
+    {
+        if (!_rows(mesh_item_id, c).empty())
+            n++;
+    }
```

Comment:
the function will be called for each element within LocalDataInitializer, and not used during time stepping.